scripts/m8.sh

```diff
@@ -18,4 +18,4 @@ touch "$LOG_1"
 touch "$LOG_2"
 
 ./demo_direct_call.sh -p 9944 -P 2000 -t first 2>&1 | tee "$LOG_1"
-./demo_direct_call.sh -p 9944 -P 2000 -t second 2>&1 | tee "$LOG_2"
+./demo_direct_call.sh -p 9944 -P 3000 -t second 2>&1 | tee "$LOG_2"
```

Comment:
The second call is done on worker 2, so we know that there was sidechain synchronization for the balances